From 586b0eb045dbfff3c5cdcd3c2a9e8949870303aa Mon Sep 17 00:00:00 2001
From: Anil Kumar Mamidala <anil.mamidala@xilinx.com>
Date: Thu, 8 Apr 2021 11:41:11 +0530
Subject: [PATCH 1790/1851] media: i2c: ap1302: Increase MIPI TCLK POST timing
 value

commit 66854fa953dc39d67a1503374d5fe9f24597ca9e from
https://github.com/Xilinx/linux-xlnx.git

The current TCLK POST values set is failing for some of the boards.
So increase the MIPI TCLK POST timing value to fix pipeline freeze
issue.

Signed-off-by: Anil Kumar Mamidala <anil.mamidala@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/i2c/ap1302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ap1302.c b/drivers/media/i2c/ap1302.c
index dd0756269098..b508dd87f0b4 100644
--- a/drivers/media/i2c/ap1302.c
+++ b/drivers/media/i2c/ap1302.c
@@ -1233,7 +1233,7 @@ static int ap1302_set_mipi_t3_clk(struct ap1302_device *ap1302)
 
 	/* Read Tclk post default setting and increment by 2 */
 	t_clk_post = ((mipi_t3 & AP1302_TCLK_POST_MASK)
-					>> AP1302_TCLK_POST_SHIFT) + 0x2;
+					>> AP1302_TCLK_POST_SHIFT) + 0x5;
 	/* Read Tclk pre default setting and increment by 1 */
 	t_clk_pre = ((mipi_t3 & AP1302_TCLK_PRE_MASK)
 					>> AP1302_TCLK_PRE_SHIFT) + 0x1;
-- 
2.31.1

